nomos-services/blend/src/core/mod.rs

```diff
@@ -504,6 +516,21 @@ where
                     membership,
                 },
         }) => {
+            let reward_session_info = reward::SessionInfo::new(
+                session,
+                current_public_inputs.leader.pol_epoch_nonce,
+                membership.size() as u64,
+                poq_core_public_inputs.quota,
+                settings.scheduler.cover.message_frequency_per_round,
+            )
+            .expect("Reward session info must be created successfully. Panicking since the service cannot continue with this session");
+            if let Some(activity_proof) =
+                blending_token_collector.rotate_session(&reward_session_info)
+            {
+                info!(target: LOG_TARGET, "Activity proof generated: {activity_proof:?}");
```

Comment:
So here what we get is the activity proof of a previous session before the one we are actually transitioning from. Will we still be able to submit activity proofs for it? Just asking for my own understanding.